src/main/resources/db/changelog/changes/create-user-tenant-table.xml

```diff
@@ -55,4 +55,10 @@
       <column name="updated_date" type="timestamp without time zone"/>
     </addColumn>
   </changeSet>
+
+  <changeSet id="MODCON-38@@add-central-tenant-id-field" author="d.butramyou">
```

Comment:
We don't need this change, we are storing centralTenantId in table consortia-configuration and in tenant table we have flag isCentral. So we just need to have centralTenantId in payload to send to mod-users central tenant, but not in mod-consortia DB

Reply:
Deleted